src/basicfuns.jl

```diff
@@ -115,8 +173,11 @@ function _log1pmx_ker(x::Float64)
 end
 
 
-## logsumexp
+"""
+    logsumexp(x::Real, y::Real)
```

Comment:
we should probably rename this `logaddexp`?

Reply:
Look a bit further down.  There is a method for `x::AbstractArray{<:Real}` for which `sum` as part of the name is appropriate.